src/ProtocolGateway.Core/Mqtt/MqttAdapter.cs

```diff
@@ -341,11 +341,9 @@ async void ProcessPendingSubscriptionChanges(IChannelHandlerContext context)
                     }
                     queue.Clear();
 
-                    if (!this.sessionState.IsTransient)
```

Comment:
>IsTransient [](start = 43, length = 11)

as it is a breaking change, pls add a point to RELEASE_NOTES.md for 3.0.0 release.